core/compose-markdown/src/main/kotlin/com/nasdroid/core/markdown/components/MarkdownBlockQuote.kt

```diff
@@ -35,35 +35,37 @@ fun MarkdownBlockQuote(
     codeBlockStyle: CodeBlockStyle,
     modifier: Modifier = Modifier,
 ) {
-    Box(
-        modifier = Modifier
-            .background(style.background, style.shape)
-            .then(modifier)
-    ) {
-        Column(
-            modifier = Modifier.padding(style.innerPadding),
-            verticalArrangement = Arrangement.spacedBy(textStyles.textStyle.fontSize.toDp())
+        Box(
+            modifier = Modifier
+                .background(style.background, style.shape)
+                .then(modifier)
         ) {
-            blockQuote.children.forEach {
-                Row(Modifier.height(IntrinsicSize.Min)) {
-                    Surface(
-                        shape = CircleShape,
-                        color = style.barColor,
-                        modifier = Modifier
-                            .padding(end = style.paddingAfterBar)
-                            .width(style.barWidth)
-                            .fillMaxHeight()
-                    ) {}
-
-                    MarkdownNode(
-                        node = it,
-                        textStyles = textStyles,
-                        textStyleModifiers = textStyleModifiers,
-                        blockQuoteStyle = style,
-                        codeBlockStyle = codeBlockStyle
-                    )
+            Row(Modifier.height(IntrinsicSize.Min)) {
+                Surface(
+                    shape = CircleShape,
+                    color = style.barColor,
+                    modifier = Modifier
+                        .padding(end = style.paddingAfterBar)
```

Comment:
This is gonna make some big padding by default (inner padding + padding after bar). Maybe we should just drop `paddingAfterBar`?

Reply:
Good catch, I ended up adding the padding since this issue was happening anyway.